pkg/kuttlctl/cmd/test.go

```diff
@@ -67,7 +67,7 @@ func newTestCmd() *cobra.Command { //nolint:gocyclo
 	options := harness.TestSuite{}
 
 	testCmd := &cobra.Command{
-		Use:   "test [flags]... [test directories]...",
+		Use:   "test [flags]... [test suite]...",
```

Comment:
Directories in kuttl have different semantic meaning, they can be a suite or a case. This uses the correct terminology so make it clear that the suite is expected here, passing a case does not work.